Allow selectively including polyfills (#30)
Richienb/node-polyfill-webpack-plugin: readme.md

Type: `object`

> The `excludeAliases` and `includeAliases` key are mutally exclusive.

Reviewer: ```suggestion
`excludeAliases` and `includeAliases` are mutually exclusive.
```

}
```

> #### includeAliases

Reviewer: Add a comment before the docs for `excludeAliases` and `includeAliases` that declares them mutually exclusive.

> Alternatively, you can chose to only include certain aliases. In this example, only `console` wil be polyfilled.

Reviewer: ```suggestion
Alternatively, you can choose to only include certain aliases. For example, you can only have `console` polyfilled.
```
